### DISCUSSION TITLE
Block NSFW Content

### DISCUSSION THREAD

**Discussion by Sand004** [carson-katri/dream-textures#201] [General]
Hi

I want to let my kids play around with the AI a bit and of course I dont want them to generate a certain kind of content ;)

Is there a way to block NSFW content? I know that normal stable diffusion has this enabeled by default.

Best Regards

**Comment by szczyglo74** [comment]
Hi
Consider training your own AI model with cats, unicorns and fairy tale characters. I'm serious. Here is tutorial how to do it: https://www.youtube.com/watch?v=7m__xadX0z0
Best regards

**Comment by TiagoTiago** [comment]
I am not familiar with the code, but I imagine it should be possible to look at the original and the changed version of the scripts in the addon and figure out what files you need to copy and what lines you need to re-add. Might be quicker to just get one of the webui versions that already have the NSFW filter option and have your kids play with that than altering the code of the addon though.
